app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/Filter.kt

```diff
@@ -138,6 +139,7 @@ fun FilterPanel(
             if (isTextFieldVisible.value) {
               SearchBarListScreen(
                   addressViewModel,
+                  parkingViewModel,
```

Comment:
Suprised the search bar didn't NEED the ViewModel in the first place

Reply:
At the beginning the search bar was used for the map which is not dependant on the parkings but rather on the Nominatim API and MapBox Map.